app/views/layouts/application.html.erb

```diff
@@ -9,36 +9,36 @@
 
```

Comment:
looks like a linter ran here and made more changes in addition to the actual required code changes, please make a PR without auto linting